### PR TITLE
gnuradio: fix --with-documentation

### PR DESCRIPTION
**Pull request by metacollin**
Fixed ruby error preventing formula from building when --with-documentation was used, and added a patch to fix wx not being found on systems with multiple versions installed.

### CODE REVIEW

File: Library/Formula/gnuradio.rb [hunk 1]
```diff
@@ -19,6 +19,14 @@ class Gnuradio < Formula
     sha256 "9e1c612f0f4063d387d85517cc420f050f49c7903d36fab45b72e8d828549e3c"
   end
 
+  # The wxpython dependency isn't properly found on multiversion 
+  # python systems.  This patches gr-wxgui's CMakeLists.txt file 
+  # to correctly import wxversion and select the brewed wxpython package.
```
Comment (dunn) [inline Library/Formula/gnuradio.rb]: Can you submit this patch upstream? Thanks!

File: Library/Formula/gnuradio.rb [hunk 2]
```diff
@@ -138,7 +147,7 @@ def install
     ENV.prepend_create_path "PYTHONPATH", libexec/"vendor/lib/python2.7/site-packages"
 
     res = %w[Markdown Cheetah lxml numpy]
-    res << %w[sphinx sphinx_rtd_theme alabaster babel docutils pygments
+    res += %w[sphinx sphinx_rtd_theme alabaster babel docutils pygments
```
Comment (bfontaine) [inline Library/Formula/gnuradio.rb]: Please leave `<<` here, it modifies `res` in-place while `+=` creates a new array then assign it to `res`.

Comment (jiegec) [inline Library/Formula/gnuradio.rb]: Nope. That means appending the whole list as a item to that list. My fault.

Comment (bfontaine) [inline Library/Formula/gnuradio.rb]: Right.

**Comment by bfontaine** [comment]
Ping re Alex’ comment?

**Comment by DomT4** [comment]
This should be resolved now. Please `brew update` and let us know if you run into any issues, Thanks!

**Pull request closed without merging**